cunumeric/settings.py

```diff
@@ -82,5 +88,71 @@ class CunumericRuntimeSettings(Settings):
         """,
     )
 
+    fast_math: EnvOnlySetting[int] = EnvOnlySetting(
+        "fast_math",
+        "CUNUMERIC_FAST_MATH",
```

Comment:
Add the default values here?

Reply:
whoops https://github.com/nv-legate/cunumeric/pull/893/commits/12699dded4ec1a93b2fc01e59dfb0cb0142646e7